docs/Steps/memory.md

```diff
@@ -81,6 +81,25 @@ Set memory value
 When I save 'value' to memory as 'key'
 ```
 
+---
+### I save multiline string to memory as \{string}:
```

Comment:
I would describe it in following way
I save multiline string to memory as \{string}:[Multiline]

Reply:
No looks like that
![image](https://github.com/qavajs/qavajs.github.io/assets/46843062/9fa4875d-abb3-42e4-be52-61e2067d9062)